packages/devops/ci/e2e.env.example

```diff
@@ -1,3 +1,6 @@
 CYPRESS_RECORD_KEY=
 CYPRESS_TEST_USER_EMAIL=
 CYPRESS_TEST_USER_PASSWORD=
+
+# Avoid day ticking over in e2e test run, see RN-803
+TZ=Australia/Melbourne
```

Comment:
Added TZ env var to the container, this is then read by cypress, see thread https://github.com/cypress-io/cypress/issues/1043#issuecomment-623293020